docs/hello-neighbour.yaml

```diff
@@ -15,6 +15,15 @@ tags:
   description: "Everything relating to church members interacting with the system"
 schemes:
 - "https"
+securityDefinitions:
+  jwtAuth:
+    type: apiKey
+    name: X-JWT
```

Comment:
JWTs are usually sent as a `Bearer` token in the `Authorization` header. While we are free to do whatever we want because our app can control the HTTP headers it sends, this is quite unconventional.